gc/base/SweepPoolManagerAddressOrderedListBase.cpp

```diff
@@ -383,9 +412,16 @@ MM_SweepPoolManagerAddressOrderedListBase::flushFinalChunk(
 					sweepState->_connectPreviousFreeEntrySize,
 					sweepState->_connectPreviousChunk->trailingFreeCandidate);
 
+			if (memoryPool->doesNeedClearCardForConnectFreeEntry()) {
```

Comment:
For this block that cannot be (at least not obviously to me) piggybacked to an existing virtual method of SweepPoolManager, we need to introduce a new one that would do nothing for base class and do this thing for VLHGC variant.

For example, method could be called connectOuterMemoryPostProcess